components/Dialogs/src/MDCAlertController.m

```diff
@@ -398,8 +398,7 @@ - (void)viewWillLayoutSubviews {
         UIUserInterfaceLayoutDirectionRightToLeft) {
       for (UIButton *button in self.actionButtons) {
         CGRect buttonRect = button.frame;
-        CGRect flippedRect = MDCRectFlippedForRTL(buttonRect,
-                                                  CGRectGetWidth(self.view.bounds),
+        CGRect flippedRect = MDCRectFlippedForRTL(buttonRect, CGRectGetWidth(self.view.bounds),
```

Comment:
I believe once you break up parameters on a line the recommended practice is to have them each on their own individual line.